HyperV/templates/interfaces.template

```diff
@@ -0,0 +1,28 @@
+# Injected by Nova on instance boot
```

Comment:
We should use this from the nova build folder rather than a hardcoded file here.